quinn-proto/src/connection/streams/state.rs

```diff
@@ -50,7 +50,7 @@ pub struct StreamsState {
     /// Streams are only added to this list when a write fails.
     pub(super) connection_blocked: Vec<StreamId>,
     /// Connection-level flow control budget dictated by the peer
-    pub(super) max_data: u64,
+    max_data: u64,
```

Comment:
This doesn't appear to be necessary (anymore)?

Reply:
What doesn't, exactly?

Reply:
`pub(super)` was removed because the variable is no longer accessed from the connection and replaced by the `write_limit` function. Not exactly sure what the comment is about otherwise.

Reply:
Ah, sorry, yes, that is great!